buildspec_combined.yaml

```diff
@@ -0,0 +1,84 @@
+version: 0.1
+component: build
+timeoutInSeconds: 20000
+runAs: root
+shell: bash
+env:
+  exportedVariables:
+   - GITHUB_MD5_HASH
+   - GITLAB_MD5_HASH
+   - BITBUCKET_CLOUD_MD5_HASH
+   - BITBUCKET_SERVER_MD5_HASH
+   - PUBLIC_GITLAB_SERVER_MD5_HASH
+   - VBS_MD5_HASH
+steps:
+  - type: Command
+    name: "Calculating MD5 Hash of Github Large Repo"
+    command: |
+      cd ${OCI_WORKSPACE_DIR}
+      zip -d TestBuildSource_GitHub.zip */.git/*
+      export GITHUB_MD5_HASH=`md5sum TestBuildSource_GitHub.zip | awk '{ print $1 }'`
+    onFailure:
```

Comment:
If the control comes to the onFailure block, the build run will not fail. In that case, what will be the value of the exported variable: GITHUB_MD5_HASH?

Reply:
GITHUB_MD5_HASH will be an empty string. The build run will not fail but we have placed assertions in the canary test to check if these exported variables are not empty